sources/packages/backend/apps/db-migrations/src/migrations/1711342815738-CreateStudentLoanBalances.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class CreateStudentLoanBalances1711342815738
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Create-student-loan-balances.sql", "StudentLoanBalances"),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Drop-student-loan-balances.sql", "StudentLoanBalances"),
```

Comment:
The file name should be `Rollback-create-student-loan-balances.sql`.

Reply:
Dont we use drop if we create a new table?

Reply:
As we talked, the latest decision was just to add a `Rollback` prefix.